pkg/report/report_test.go

```diff
@@ -290,7 +300,7 @@ func updateReportTest(t *testing.T, test, parsed *ParseTest) {
 	buf.Write(parsed.Headers(test.Frame != ""))
 	fmt.Fprintf(buf, "\n%s", test.Log)
 	if test.HasReport {
-		fmt.Fprintf(buf, "REPORT:\n%s", test.Report)
+		fmt.Fprintf(buf, "REPORT:\n%s", parsed.Report)
```

Comment:
I don't understand neither old logic nor the change. Could you please explain the motivation for this change?
Is this the intentional or accidental change?